American Whitewater/Etc/Segues.swift

```diff
@@ -7,7 +7,8 @@ enum Segue: String {
     case alreadyOnboarded
     case onboardingCompleted
     case articleDetail
-    case gageDetail
+    case gaugeDetail
```

Comment:
Minor: Just fyi, we had settled on "gage" as the spelling in this repo for consistency (mostly because it's shorter). Both "gage" and "gauge" are correct in english.

Reply:
Ya, I'm going to have to refactor this pretty heavily. I thought it was a typo on the original codes part. I didn't understand the difference at the time. I'll work on it.